iothub/service/samples/PnpServiceSamples/noComponentSample/Microsoft.Azure.Devices.PnpServiceSample.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\src\Microsoft.Azure.Devices.csproj" />
```

Comment:
Make a note to update this to package reference, once we release the preview nuget

Reply:
will do, added in my todo list post this PR.